modules/core/src/main/scala/sangria/execution/deferred/FetcherBasedDeferredResolver.scala

```diff
@@ -182,6 +182,34 @@ class FetcherBasedDeferredResolver[-Ctx](
     }
   }
 
+  private def resolveConcurrentBatches(
+    ctx: FetcherContext[Ctx] @uncheckedVariance,
+    f: Fetcher[Ctx, Any, Any, Any] @uncheckedVariance,
+    nonCachedIds: Vector[Any]
+  )(implicit ec: ExecutionContext): Iterator[Future[(Vector[Any], Try[Seq[Any]])]] = {
+    val groupedIds: Iterator[Vector[Any]] = ctx.fetcher.config.maxBatchSizeConfig match {
+      case Some(size) ⇒ nonCachedIds.grouped(size)
+      case None ⇒ Iterator(nonCachedIds)
+    }
+
+    val groupedBatches: Iterator[TraversableOnce[Vector[Any]]] = ctx.fetcher.config.maxConcurrentBatchesConfig match {
+      case Some(size) ⇒ groupedIds.grouped(size)
+      case None ⇒ Iterator(groupedIds)
```

Comment:
We are now using the `=>`. Can we change them?

Reply:
Replaced! I also updated `.scalafmt.conf` so no one has to ever think about it!